portfolio/src/main/webapp/script.js

```diff
@@ -85,3 +81,26 @@ function convertTime(timestamp) {
     return formattedTime;
 }
 
+/**
+ * This function is to get data from "/data" Servlet and fetch the latest data every 3 seconds 
+ */
+
+var getting = {
+
+        url:'/data',
+        dataType:'json',
+        success:function(res) {
+
+        const commentsElement = document.getElementById('comment-list');
```

Comment:
How about putting this part in a common function that's shared between getDataFromDataServlet and this function?

Reply:
Yes sure, will try it